dspace/config/dspace.cfg

```diff
@@ -938,6 +938,9 @@ cc.submit.setname = true
 # Store license bitstream (RDF license text) during web submission
 cc.submit.addbitstream = true
 
+# ONLY JSPUI, enable Creative Commons admin 
+webui.submit.enable-cc = false
```

Comment:
is it a new feature? or something that was previously missing in the configuration?